apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```
@@ -767,6 +796,105 @@ export const IndictmentCount: FC<Props> = ({
+                    )
+
+                    updateIndictmentCountState(
+                      indictmentCount.id,
+                      { recordedSpeed },
+                      setWorkingCase,
+                    )
+                  }}
+                  onBlur={(event) => {
+                    const recordedSpeed = parseInt(event.target.value)
+
+                    if (isNaN(recordedSpeed)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)